index.html

```diff
@@ -15,255 +15,261 @@
-            class="btn button_decoration"
-            value="Start collaboration"
-          />
-        </fieldset>
-      </form>
-      <!-- -->
-      <footer>
-        <a href="#" class="resume">
-          <p>Get my resume</p>
-        </a>
-        <ul class="social">
-          <li class="socialicon">
-            <a href="https://stackoverflow.com/users/17562225/mukhammadrizooff"
-              ><img src="icons/ic_stackoverflow.png" alt="stackoverflow"
-            /></a>
-          </li>
-          <li class="socialicon">
-            <a href="https://github.com/mukhammadrizooff"
-              ><img src="icons/ic_github.png" alt="Github"
-            /></a>
-          </li>
-          <li class="socialicon">
-            <a href="#"><img src="icons/ic_twitter.png" alt="twitter" /></a>
+      <div class="container-about">
+        <h2 class="aboutheader">About</h2>
+        <p class="introabout">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut,
+        </p>
+        <button type="button" class="btn2">Get my CV</button>
+      </div>
+      <div class="container-about2">
+        <ul class="grid-container">
+          <li class="aboutcard">
+            <ul class="grid-item1">
+              <li class="about-title">
+                <h3>Languages</h3>
+              </li>
+              <li>JavaScript (ES5/ES6)</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
           </li>
-          <li class="socialicon">
-            <a href="https://www.linkedin.com/in/mukhammadrizooff"
-              ><img src="icons/ic_linkedin.png" alt="linkedin"
-            /></a>
+          <li class="aboutcard">
+            <ul class="grid-item2">
+              <li class="about-title">
+                <h3>Frameworks</h3>
+              </li>
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capibara</li>
+              <li>Selenium</li>
+            </ul>
           </li>
-          <li class="socialicon">
-            <a href="https://medium.com/@mukhammadrizooff"
-              ><img src="icons/ic_medium.png" alt="Medium"
-            /></a>
+          <li class="aboutcard">
+            <ul class="grid-item3">
+              <li class="about-title">
+                <h3>Skills</h3>
+              </li>
+              <li>Codekit</li>
+              <li>Codepen</li>
+              <li>Github</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+            </ul>
           </li>
-          <li class="socialicon">
-            <a href="#"><img src="icons/ic_angellist.png" alt="Angelist" /></a>
+          <li class="aboutcard">
+            <ul class="grid-item4">
+              <li><img src="images/dots.png" width="170" alt="dots" /></li>
+            </ul>
           </li>
         </ul>
```

Comment:
- [ ] I think it could be a good idea to center the text of each skills card because according to the template these should be in the middle of the card. Please, address this issue to match the Figma design as it is required :smiley: :pushpin:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144728408-a6edd7ad-cfff-4203-9ac8-c9cc3942a2a3.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144728423-73ee723c-2613-4648-99a0-a8fcb9509c95.png)

Reply:
Not Done

Reply:
Ok